pkg/session/session.go

```diff
@@ -2355,6 +2376,10 @@ func queryFailDumpTriggerCheck(config *traceevent.DumpTriggerConfig) bool {
 	return config.Event.Type == "query_fail"
 }
 
+func isInternalDumpTriggerCheck(config *traceevent.DumpTriggerConfig) bool {
+	return config.Event.Type == "is_internal"
```

Comment:
why the event type is called `is_internal` which more like a flag itself, not just `internal`

Reply:
OK, now I change it to `dump_trigger.suspicious_event.is_internal = true | false` from the old setting `dump_trigger.suspicious_event = is_internal` @D3Hunter